shared/src/commonMain/kotlin/com/vml/tutorial/plantshop/basket/presentation/BasketScreenState.kt

```diff
@@ -0,0 +1,25 @@
+package com.vml.tutorial.plantshop.basket.presentation
+
+import com.vml.tutorial.plantshop.core.utils.exts.orFalse
+import com.vml.tutorial.plantshop.core.utils.exts.orZero
+import com.vml.tutorial.plantshop.plants.domain.Plant
+
+data class BasketScreenState(
+    val items: List<BasketItemState>? = null,
+    val discount: Double = 3.0   // 3 USD
+) {
+    private val itemsPrice: Double get() = items.orEmpty().sumOf { it.totalPrice }
```

Comment:
At times, the sum contains an excessive number of digits, so it would be nice to round the sum :) (For example the total price of 3 Peace Lilies and 2 Sunflowers)

Reply:
I added an extension function "roundTo" so I could eliminate those digits, I hope you meant by that :D